act-framework.bs

```diff
@@ -138,8 +138,86 @@ Editor note: This section describes the required properties of the data output b
 Rule Aggregation {#output-aggregation}
 --------------------------------------
 
-Editor note: In this section we describe how the data that is returned from a rule can be combined to give a higher level view of the conformance to accessibility requirements. Rules provide very low level information, this is valuable for people working at that level, but managing accessibility of products as a whole requires a higher level understanding of the accessibility.
-
+As describe in section [ACT Data Format](#output-format) a rule will return a list of results, each of which contain 1) an outcome (passed, failed or cantTell), 2) the selected item, 3) the Rule ID and 4) the test subject. We can create a new result, that combines all the results of that rule, to create provide a single result for that rule.
```

Comment:
should be "that combines all the results of that rule, to create a single result for that rule."